src/generate-critical-css.ts

```diff
@@ -36,8 +36,12 @@ async function collateCssFiles(
 			// Convert relative URLs to absolute.
 			const relativeUrls = Object.keys( cssIncludes );
 			const absoluteIncludes = relativeUrls.reduce( ( set, relative ) => {
-				const absolute = new URL( relative, url ).toString();
-				set[ absolute ] = cssIncludes[ relative ];
+				try {
+					const absolute = new URL( relative, url ).toString();
+					set[ absolute ] = cssIncludes[ relative ];
+				} catch ( err ) {
+					// Ignore invalid URLs.
```

Comment:
I feel this should log a warning that a URL is invalid, with the URL that is being rejected. It'll help us debug future issues. :)

Reply:
Log added. Do you mean disabling eslint because of the `console.log`, or am I missing some other lint whinings?